### DISCUSSION TITLE
deploy skywalking-ui by kubernetes but found exception

### DISCUSSION THREAD

**Discussion by chunhaiyangzhili** [apache/skywalking#10391] [Q&A] [answered]
exception log:
![image](https://user-images.githubusercontent.com/35128699/218666546-36a4a216-c623-4755-817b-db1de18f5505.png)

**Comment by chunhaiyangzhili** [comment]
hope someboy can help me

**Comment by kezhenxu94** [comment, marked as answer]
Please share how did you deploy this. The exception means the WebApp doesn't have a correct URL to OAP, you need to add http:// prefix

**Reply by chunhaiyangzhili** [reply]
thanks a lot!!!
I solve the problem，I see the page from skywalking!
thank you again!!!
good people have long live!!! hhh...
![image](https://user-images.githubusercontent.com/35128699/218738571-5dba769b-9bf4-4092-9e16-f5d925526cb2.png)
